src/routes/(dashboard)/ManagerDashboard.svelte

```diff
@@ -223,7 +223,7 @@
 
     const assignEditor = async (contentIds: number[]) => {
         if (contentIds.length > 0) {
-            await postToApi<null>('/resources/content/assign-editor', {
+            await postToApi<null>('/resources/content/send-for-editor-review', {
```

Comment:
Is this going to break any flows around company review? I was thinking the previous endpoint updated the status to company review when assigning to another manager, didn't know how this current endpoint works.

Reply:
Sure here's what happened to the assign-editor endpoint (once it's merged and deployed)
It has been split into 
- `send-for-editor-review` (assign-editor still exists but has been marked for deletion)
  - Resource content is New/AIDraftComplete and is sent to status EditorReview (and assigned if needed, see Nate's new changes because he set things to where they're assigned at the time of translation)
  - Resource content is in EditorReview already and assigned to someone else, no status change
  - Resource content is in CompanyReview and assigned to someone else, no status change
- `pull-from-publisher-review`
  - Resource content is in PublisherReview or NotApplicable and is pulled/sent back to CompanyReview
- `pull-from-review-pending`
  - Resource Content is in ReviewPending and is pulled/sent back to CompanyReview
  
 You can refer to the [assign-editor refactor ticket](https://linear.app/biblionexus/issue/BIB-789/refactor-assign-editor-endpoint) and to the [aquifer-server pr](https://github.com/BiblioNexusStudio/aquifer-server/pull/527)
 
I was thinking it wouldn't be a good idea to merge/deploy the aquifer-server side until this is reviewed and approved since they're dependent on each other. The aquifer-server PR is already approved but another pair of eyes on it would be appreciated too